### PR TITLE
Server

### PR DESCRIPTION
**Pull request by MonicaCupp**
@vpatel89  @sbirvin1s 

Changes the password issue
Adds a few queries

### CODE REVIEW

**Review by vpatel89 — APPROVED** [review]

File: db.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
+const { USER, HOST, DATABASE, PASSWORD } = process.env;
+
+
 const pool = new Pool ({
-  user: "q_and_a_service",
-  host: "localhost",
-  database: "q_and_a_service",
-  password: "#sdc#",
-  port: 5432
+  user: USER,
```
Comment (sbirvin1s) [inline db.js:8]: Just a note. When using `USER` here I ran into a conflict where the value for `USER` defined in my `.env` file was getting overwritten by the local system's value for `USER`.

To get around this, I changed the variable to something like `DB_USER` which avoided the conflicting variables and prevented the value from being over written.

Not sure if this is something you will run into, but it is something to watch out for

File: src/q_and_a_service/controller.js [hunk 2]
```diff
@@ -1,14 +1,39 @@
 const pool = require ('../../db.js');
 const queries = require ('./queries.js');
 
-
 const getQA = (req, res) => {
-  pool.query(queries.getQA , (error, results) => {
+  console.log('queries.qa', queries.getQA);
```
Comment (sbirvin1s) [inline src/q_and_a_service/controller.js:5]: Is this `console.log` still needed? If not, you might consider removing it

**Review by sbirvin1s — APPROVED** [review]
Looks good! 👍

File: db.js (hunk 1 above)
Comment (MonicaCupp) [reply db.js:8]: Thanks for this!  I will make the change just to be on the safe side.

File: src/q_and_a_service/controller.js (hunk 2 above)
Comment (MonicaCupp) [reply src/q_and_a_service/controller.js:5]: You are right.  I will delete it.

**Pull request merged**